src/stateProviders/provideLocalSqon.js

```diff
@@ -0,0 +1,15 @@
+import { provideState } from 'freactal';
+
+export default provideState({
+  initialState: props => ({
+    advancedFacetSqon: null,
```

Comment:
These shouldn't be specific to the advanced facets, make it something like localSQON. 
It would probably also initialize to props.sqon.